### ISSUE TITLE
https://seekingalpha.com/market-outlook

### ISSUE THREAD

**Issue by Left-Banker** [reek/anti-adblock-killer#3391]
Does not work on any but the homepage.

Chrome, Firefox or Opera

I had it working fine on a Win 7 box a Macbook Pro. On a new Win10 machine it will not work on any browser for this site anywhere but on the home page

![image](https://user-images.githubusercontent.com/30219115/28637838-ba2d6830-71f7-11e7-823f-8c756d2a4752.png)

**Comment by uBlock-user** [comment]
Can't reproduce.

**Comment by uBlock-user** [comment]
@jspenguin2017 can you ?

**Comment by jspenguin2017** [comment]
I see this in EasyList, which white lists a bidding script: `@@||seekingalpha.com^$script`

**Comment by uBlock-user** [comment]
Yeah I blocked it, cleared cookies and opened the page into a new tab and still can't reproduce it.

**Comment by Left-Banker** [comment]
I cannot understand why you cannot reproduce it. When I wrote this all was good on a MacBook and a Win 7 machine but I was having the issue on a new  Win 10 laptop (as noted). Now it fails on every one of the machines.

I've clear cookies. I closed the browser. I've tried multiple browsers (Chrome, Firefox, Opera). It never works. 

I hate to keep bugging you about this, but either I'm doing  something very wrong (which I doubt, but it's not impossible) or SeekingAlpha has gotten a lot better at defeating this.

**Comment by jspenguin2017** [comment]
I still can't reproduce it.

**Comment by Left-Banker** [comment]
I think I've got a reason. Apparently the site is targeting users at different levels: 

----
"15% of SA users who currently use an ad blocker will see one of the following:

   
   
   - 5% will see an option to whitelist SA. They will be able to shut down the call-to-action and continue reading if they choose.
   - 5% will be asked to whitelist SA, and will not be able to shut down the ad-blocker-blocker without whitelisting SA.
   - 5% will be given the option to whitelist SA, or pay $9.99/month to keep using their ad blocker."
I'm guessing there's not much help in this case but thanks for your efforts.
-----

On Tuesday, August 15, 2017, 4:01:15 PM PDT, jspenguin2017 <notifications@github.com> wrote:


I still can't reproduce it.

**Comment by jspenguin2017** [comment]
Well, if it is bonded to accounts on server side, then it's hard to debug. I will need a test account then.

**Comment by jspenguin2017** [comment]
Try uBlock Protector, if that doesn't work, open a confidential issue here: https://gitlab.com/xuhaiyang1234/uBlockProtectorSecretIssues/issues
I will have a look.
